diff -uraN flatpak-1.15.7/common/meson.build omv-flatpak-1.15.7/common/meson.build
--- flatpak-1.15.7/common/meson.build	2024-03-27 15:40:30.000000000 +0100
+++ omv-flatpak-1.15.7/common/meson.build	2024-03-31 19:08:04.070974138 +0200
@@ -253,7 +253,7 @@
   gnu_symbol_visibility : 'hidden',
   include_directories : [common_include_directories],
   install : true,
-  link_args : ['-export-dynamic'],
+  link_args : ['-Wl,--export-dynamic'],
   link_whole : [
     libflatpak_common_base,
     libflatpak_common,
